File: w09/d03/jimmy_davis/mail_objects_app/spec/javascripts/postcard_spec.js
describe("a postcard", function(){

 it("has a message string", function() {
   postcard = new Postcard();
   postcard.writeMessage("Greetings from NYC!");
   var test = function() {
     postcard.writeMessage(123);
   };
   expect(postcard.message).toEqual("Greetings from NYC!");
   expect(test).toThrow();
  });

 });

File: w09/d01/joshua_oynick/mail_items/spec/postcard_spec.rb
require_relative 'spec_helper'
require_relative '../lib/postcard'
require_relative '../lib/mailitem'
require_relative '../lib/package'

describe Postcard do

  it "inherits a mail_item" do
    expect(Postcard.superclass).to eq(Mailitem)
  end

  it "has a message" do
    card = Postcard.new(message: "bambam")
    expect(card.message).to eq 'bambam'
    expect(card).to respond_to(:message)
  end


end
